src/Features/LanguageServer/Protocol/Handler/Breakpoints/ValidateBreakableRangeHandler.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+using System.Runtime.Serialization;
+using Microsoft.CodeAnalysis.Debugging;
+
+namespace Microsoft.VisualStudio.LanguageServer.Protocol
+{
+    [DataContract]
+    internal sealed class ValidateBreakableRangeParams
+    {
+        [DataMember(Name = "textDocument")]
+        public TextDocumentIdentifier TextDocument { get; }
+
+        [DataMember(Name = "range")]
+        public Range Range { get; }
+
+        public ValidateBreakableRangeParams(TextDocumentIdentifier textDocument, Range range)
+        {
+            TextDocument = textDocument;
+            Range = range;
+        }
+    }
+}
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler
+{
+    [ExportRoslynLanguagesLspRequestHandlerProvider, Shared]
+    [ProvidesMethod("textDocument/validateBreakableRange")]
+    internal sealed class ValidateBreakableRangeHandler : AbstractStatelessRequestHandler<LSP.ValidateBreakableRangeParams, LSP.Range?>
```

Comment:
would be good to add a test that at least validates that we can call this and get a response.  Probably not much more needed here assuming we already have test cases for the IBreakpointResolutionService

Reply:
Added tests